spec/unit/berkshelf/berkshelf/api_client/chef_server_connection_spec.rb

```diff
@@ -0,0 +1,67 @@
+require "spec_helper"
+
+describe Berkshelf::APIClient::ChefServerConnection do
+  let(:instance) do
+    described_class.new(
+      {
+        server_url: "https://chef.example.org/organizations/foobar",
+        timeout: 30,
+        open_timeout: 3,
+        ssl: {},
+      }
```

Comment:
this looks buggy, you don't want the surrounding hash

Reply:
Fixed